examples/convert_little_endian_to_big.cpp

```diff
@@ -5,7 +5,7 @@
 
 #include <iostream>
 
-#include <endian/convert_endian.hpp>
+#include <endian/little_endian.hpp>
```

Comment:
I think we should move the `is_big_endian` static function to its own header. It might be useful in other projects and tests.